text/0000-safer-transmute.md

```diff
@@ -532,15 +408,28 @@ where
 
 In the above definitions, `Src` represents the source type of the transmutation, `Dst` represents the destination type of the transmutation, and `Neglect` is a parameter that [encodes][options] which static checks the compiler ought to neglect when considering if a transmutation is valid. The default value of `Neglect` is `()`, which reflects that, by default, the compiler does not neglect *any* static checks.
 
+The transmute! macro provides a shorthand for safely transmuting a value:
+```rust
+pub macro transmute($expr: expr) {
+    core::convert::transmute::TransmuteFrom::<_>::transmute_from($expr)
+    //              ┯
+    //              ┕ the destination type of the transmute (`_` used to infer the type from context)
+}
+```
+
+A `differing_sizes` lint warns when the source and destination types of a transmutation (conducted via `transmute!` or `transmute_from`) have different sizes.
+
+
 ### Neglecting Static Checks
 [options]: #Neglecting-Static-Checks
 
-The default value of the `Neglect` parameter, `()`, statically forbids transmutes that are ill-defined or unsafe. However, you may explicitly opt-out of some static checks; e.g.:
+The default value of the `Neglect` parameter, `()`, statically forbids transmutes that are ill-defined or unsafe. However, you may explicitly opt-out of some static checks; namely:
 
 | Transmute Option    | Usable With                                             |
 |---------------------|---------------------------------------------------------|
 | `NeglectAlignment`  | `unsafe_transmute_{from,into}`                          |
 | `NeglectValidity`   | `unsafe_transmute_{from,into}`                          |
+| `NeglectSafety`     | `unsafe_transmute_{from,into}`                          |
```

Comment:
The confusing part about this naming is that any of the `Neglect` makes things unsafe (I assume), but only one of them is called `NeglectSafety`. Maybe `NeglectMuckable` or so would work better?